archive/Gcl_Archive/Associate-Cloud-Engineer/Set-Up-an-App-Dev-Environment-on-Google-Cloud.md

```
@@ -0,0 +1,540 @@
+| 扱う技術要素 | Cloud Storage / Identity and Access Management（IAM）/ Cloud Functions（Cloud Run functions）/ Pub/Sub |
+| 最終更新 | 2026-07-01 |
+
+---
+
+## 目次
+
+1. [このガイドについて](#1-このガイドについて)
+2. [コース全体像とラーニングパス](#2-コース全体像とラーニングパス)
+3. [Cloud Storage — オブジェクトストレージの基礎](#3-cloud-storage--オブジェクトストレージの基礎)
+4. [Cloud IAM — アクセス制御の基礎](#4-cloud-iam--アクセス制御の基礎)
+5. [Cloud Functions（Cloud Run functions）— イベント駆動サーバーレス](#5-cloud-functionscloud-run-functions--イベント駆動サーバーレス)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)